test/e2e/tests/confirmations/alerts/queued-confirmations.spec.ts

```diff
@@ -392,7 +392,7 @@ async function connectToDappTwoAndSwitchBackToOne(
   await driver.waitUntilXWindowHandles(4);
   await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
 
-  await driver.clickElement({
```

Comment:
race condition here, we need to wait for the dialog to close, before proceeding to the dapp 1 window